libvast/native-plugins/segment_store.cpp

```diff
@@ -212,6 +213,12 @@ system::store_actor::behavior_type passive_local_store(
           rp.delegate(static_cast<system::store_actor>(self), atom::erase_v,
                       std::move(ids));
         }
+        auto startup_duration = std::chrono::steady_clock::now() - start;
+        self->send(self->state.accountant, "passive-store.startup.runtime",
```

Comment:
Can you add this to monitor.md in the docs?

IIRC we were also unsure about the naming, probably something for @mavam.

Reply:
I think I'm too far away to have a useful opinion on this. 😉

Reply:
It's a metric that measures the duration from spawning a store to it being able to answer the first query. You're usually good at naming things, that's why I thought of you, and Thomas and I were pairing on this and neither of us came up with a good name, so `startup.runtime` is what we've got now. 😅

Reply:
good catch.

Reply:
To be frank, `startup.runtime` sounds like an oxymoron. If the semantics are "time until things are ready", then here are some suggestions:

- `startup.time`
- `startup.latency`
- `startup.duration`
- `startup.boot-time`
- `startup.load-time`
- `startup.warmup`

HTH

Reply:
What about `load.duration`? I find `startup` to be somewhat imprecise as it's not entirely clear what "startup" for a store needs.

Reply:
@dispanser and I agreed in a huddle that `init.runtime` is conveying the semantics best.

Reply:
we settled for `init.runtime` as we're initializing the store, and that word clearly conveys that this is a one-time operation (per store).